### PR TITLE
Add vets and specializations

### PR DESCRIPTION
**Pull request by SamTush**
## What does this PR do?

This PR is raised to implement the following features: 👍 :

- [x]  Add vets and specializations table
- [x] Insert more data into the vets and specializations table 

## Description of the task to be completed ↓↓ 

- Insert more data into the table

## Screenshots of the queries.

- Who was the last animal seen by William Tatcher?

Screenshot 2023-05-13 at 12 15 04

- How many different animals did Stephanie Mendez see?

Screenshot 2023-05-13 at 12 23 50

- List all vets and their specialties, including vets with no specialties.

Screenshot 2023-05-13 at 12 33 19

- List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
Screenshot 2023-05-13 at 12 49 42

- What animal has the most visits to vets?

Screenshot 2023-05-13 at 12 58 30

- Who was Maisy Smith's first visit?

Screenshot 2023-05-13 at 13 03 03

- Details for most recent visit: animal information, vet information, and date of visit.

Screenshot 2023-05-13 at 13 08 39

- How many visits were with a vet that did not specialize in that animal's species?

Screenshot 2023-05-13 at 13 10 28

- What specialty should Maisy Smith consider getting? Look for the species she gets the most.

Screenshot 2023-05-13 at 13 29 26

#### Thanks for reviewing my PR 😊

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -75,3 +75,32 @@ SET owner_id =
         WHEN name IN ('Angemon', 'Boarmon') THEN 5
         ELSE NULL
      END);
+
+
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES ('William Tatcher', 40, '2000-4-23'),('Maisy', 26, '2019-1-17'),('Stephanie Mendez', 64, '1981-5-4'),('Jack Harkness', 38, '2008-6-8');
```
Comment (alexander16108) [inline data.sql]: please change the value of the age for William Tatcher to the correct value 👍🏾

File: schema.sql [hunk 2]
```diff
@@ -27,3 +27,26 @@ ALTER TABLE animals DROP COLUMN IF EXISTS species;
...
 
 ALTER TABLE animals ADD COLUMN owner_id INTEGER REFERENCES owners(id);
+
+
+-- vet clinic database: Add 'join table' for visits
+CREATE TABLE vets (   
+id SERIAL PRIMARY KEY, 
+name VARCHAR(150),
+age INTEGER, 
+date_of_graduation DATE);
```
Comment (alexander16108) [inline schema.sql:37]: optional- its advisable to make to the value of your id not null 😄

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @SamTush , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- add screenshots 👍🏾 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @SamTush , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**